packages/klevu-ui/src/components/klevu-facet-list/klevu-facet-list.tsx

```diff
@@ -126,8 +126,7 @@ export class KlevuFacetList {
     if (!this.shouldUpdateUrlForFacets) {
       return
     }
-    const urlSearchParams = new URLSearchParams(window.location.search)
-    this.manager.readFromURLParams(urlSearchParams)
+    this.manager.readFromURLParams(new URL(document.URL).searchParams)
```

Comment:
would like to understand more, what difference is in using
window.location.search
vs
new URL(document.URL).searchParams

Reply:
It's pretty much the same, but with `new URL(document.URL)` we are relying on browser to come with `URLSearchParams` instead of us building it.